core/tests/test_trezor.wire.thp.cpace.py

```
@@ -0,0 +1,73 @@
+            "4c9c95bca3508c24b1d0b1559c83ef5b04445cc4581c8e86d8224eddd09f11d7",
+            "993c6ad11c4c29da9a56f7691fd0ff8d732e49de6250b6c2e80003ff4629a175",
+        ),
+    }
+
+    def test_compute_shared_secret(self):
+        ctx = cpace.Cpace(b"")
+        s = unhexlify(
+            "af46e36bf0527c9d3b16154b82465edd62144c0ac1fc5a18506a2244ba449aff"
+        )
+        ctx.trezor_private_key = s
+        for _, input in self.vectors_raise:
```

Ruff (0.15.21)

[error] Variable `input` is shadowing a Python builtin

(A001)